app/src/main/java/org/mozilla/fenix/library/bookmarks/viewholders/BookmarkSeparatorViewHolder.kt

```diff
@@ -25,6 +25,6 @@ class BookmarkSeparatorViewHolder(
     ) {
         this.item = item
         containerView.displayAs(LibrarySiteItemView.ItemType.SEPARATOR)
-        setupMenu(item)
+        super.bind(item, mode)
```

Comment:
If you want to keep the super method, I think this should be the first line of the function for consistency.